models/node.js

```diff
@@ -2,29 +2,67 @@ import { formatPercent } from '@/utils/string';
 import { NODE_ROLES } from '@/config/labels-annotations.js';
 
 export default {
+  availableActions() {
+    const cordon = {
+      action:     'cordon',
+      enabled:    this.isWorker && !this.isCordoned,
+      icon:       'icon icon-fw icon-pause',
+      label:      'Cordon',
+      total:      1,
+      bulkable:   true
+    };
+
+    const uncordon = {
+      action:     'uncordon',
+      enabled:    this.isWorker && this.isCordoned,
+      icon:       'icon icon-fw icon-play',
+      label:      'Uncordon',
+      total:      1,
+      bulkable:   true
+    };
+
+    return [
+      cordon,
+      uncordon,
+      ...this._standardActions
+    ];
+  },
+
+  showDetailStateBadge() {
+    return true;
+  },
+
   name() {
     return this.metadata.name;
   },
 
+  internalIp() {
+    return this.columns['internal-ip'];
+  },
+
+  externalIp() {
+    return this.columns['external-ip'];
+  },
+
   isWorker() {
     const { WORKER: worker } = NODE_ROLES;
     const labels = this.metadata?.labels;
 
-    return labels[worker] && labels[worker].toLowerCase() === 'true';
+    return (typeof labels[worker] !== 'undefined') && labels[worker].toLowerCase() === 'true';
```

Comment:
I don't know that the label is ever `"false"` or a false-y value that would ever happen either way, but these still have at least 2 conditions that would make it blow up... if you want to catch both:

```javascript
   const labels = this.metadata?.labels || {};  // Labels might not be set at all
   return `${labels[worker]}` === 'true';  // The label might be a non-string value
```

Reply:
Done.